pallets/identity/Cargo.toml

```diff
@@ -41,21 +41,24 @@ version = '2.0.0'
 default-features = false
 git = 'https://github.com/paritytech/substrate.git'
 optional = true
-tag = 'monthly-2021-09+1'
+tag = 'monthly-2022-02'
 version = '4.0.0-dev'
 
 [dependencies.frame-support]
 default-features = false
 git = 'https://github.com/paritytech/substrate.git'
-tag = 'monthly-2021-09+1'
+tag = 'monthly-2022-02'
 version = '4.0.0-dev'
 
 [dependencies.frame-system]
 default-features = false
 git = 'https://github.com/paritytech/substrate.git'
-tag = 'monthly-2021-09+1'
+tag = 'monthly-2022-02'
 version = '4.0.0-dev'
 
+[dependencies]
+scale-info = { version = "1.0.0", default-features = false, features = ["derive"] }
```

Comment:
```suggestion
scale-info = { version = "2.0.0", default-features = false, features = ["derive"] }
```

Reply:
scale-info might be why the check is currently failing

Reply:
And I just bumped version on last commit.